README.md

```diff
@@ -545,6 +546,147 @@ module.export = function migrateData(app) {
 }
 ```
 
+# Force Id
+
+Learn more about LoopBack's [forceId](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
```

Comment:
Maybe add one sentence: 
Briefly, in loopback `forceId` means user can specify the value of the primary key when creating a model instance, instead of using an auto-generated one.